symbols/golf_pin.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg version="1.1"
+    id="golf_pin"
+    xmlns:svg="http://www.w3.org/2000/svg"
+    xmlns="http://www.w3.org/2000/svg"
+    width="9"
+    height="13">
+   <path d="M 4 1 L 4 10.773438 A 2.107161 0.75 0 0 0 2.3925781 11.5 A 2.107161 0.75 0 0 0 4.5 12.25 A 2.107161 0.75 0 0 0 6.6074219 11.5 A 2.107161 0.75 0 0 0 5 10.773438 L 5 4 L 9 2.5566406 L 5 1 L 4 1 z"/>
```

Comment:
needs review for pixel alignment

Reply:
Any guidelines on pixel alignment? Here is the SVG with a 1px grid overlaid:

![image](https://user-images.githubusercontent.com/1396120/131420017-65b32755-920a-4633-b4e2-1951d0b136c1.png)